frontend/src/components/Psychologist/Profile/InactiveProfile.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { TextInput, RadioGroup, Radio, Button, Alert } from '@dataesr/react-dsfr';
+import agent from 'services/agent';
+import Page from 'components/Page/Page';
+
+const InactiveProfile = () => {
+  const [reason, setReason] = useState();
+  const [displayReason, setDisplayReason] = useState(false);
+  const [otherReason, setOtherReason] = useState('');
+  const [done, setDone] = useState(false);
+  const { token } = useParams();
+
+  const getReason = () => (reason === 'other' ? `Autre: ${otherReason}` : reason);
+
+  const selectReason = value => {
+    if (value !== 'other') {
+      setDisplayReason(false);
+      setReason(value);
+    }
+  };
+  const canValidate = reason
+  && (reason !== 'other' || otherReason.trim() !== '');
+
+  return (
+    <Page
+      background="blue"
+    >
+      {done && (
+        <Alert
+          type="success"
+          title="Vos informations ne sont plus visibles sur l'annuaire"
+          description="Vous pouvez redevenir visible à tout moment sur votre espace psychologue"
+        />
+      )}
+      <RadioGroup
```

Comment:
Why not reuse the suspension profile component?

Reply:
because we have more option here 
like connexion issues